body.tex

```diff
@@ -0,0 +1,442 @@
+\end{itemize}
+
+This can be done with the following changes in lsstsw package.
+
+
+\subsubsection{Update deploy Command} \label{sec:deploy}
+
+The \textbf{deploy} lsstsw command should accept the environment reference \textbf{envref}  as an additional command line option.
+
+The default behavior should be the same as now. If no envref is provided, the usual fixed hash shall be deployed. 
+
+In case the envref is \textit{master}, it shall deploy the latest master, if not already deployed.
+In this way, this option can be used to check if there is a newer master version of the conda environment and deploy it.
+
+The environment shall have a unique name, not the fixed one that is currently used.
+
+
+\subsubsection{Update rebuild Command} \label{sec:rebuild}
+
+The \textbf{rebuild} command should record the environment reference used for each build.
+
+This will permit in the future to retrieve the information, in order to manually activate the same conda environment before running a build.
+
+The \textbf{rebuild} command could also raise a warning, in case the active environment is different from the one previously used.
+
+
+\subsubsection{New envconfig Command} \label{sec:envconfig}
+
+A new command shall be added, that permits to choose which environment to activate.
+
+It shall accept one of following options:
+
+\begin{itemize}
+\item a \textit{scipipe\_conda\_env} environment reference to activate
+\item an existing build number, in order to activate the environment that has been used for that build.
+\item list the available environments, ready to be activated
+\item compare the active environment with an other one, a \textit{scipipe\_conda\_env} reference or a deployed environment
+\end{itemize}
+
+If no option is provided, the default reference shall be activated, in order to keep the current behavior.
+
+
+\subsection{Changes to lsst/lsst Git Package} \label{sec:newinstall}
+
+The distribution script \textit{newinstall.sh} should be updated in order to be able to deploy and activate the environment corresponding to the EUPS build that is going to be installed.
+
+This imply that the generate scripts \textit{loadLSST.*} shall accept an environment parameter as input, and deploy the environment reference before activating it, if not already deployed.
+
+
+\subsection{Changes to Jenkins builds} \label{sec:cibuilds}
+
+The Jenkins jobs, for example \textit{run-rebuild}, shall be updated in order to accept an environment reference as an optional parameter.
+In this way it will be possible to verify that the science pipelines builds with a new environment.
+
+
+\newpage
+\section{Following activities} \label{sec:follows}
+
+The changes proposed in the previous section \ref{sec:proposed} can be implemented with very minimum impact to the day by day activities.
+
+The new functionalities introduced, can facilitate already the management of the conda environment as it is done now, and the development activities in general.
```

Comment:
Remove both commas.  "facilitate already" → "already facilitate".  Remove "the" before "development".

Reply:
I removed a bit more